components/Markdown.css

```diff
@@ -45,8 +45,10 @@
   }
 
   pre:has(code) {
+    margin-left: 1rem;
+    margin-right: 1rem;
```

Comment:
**Question:** I quite like the look of code blocks filling the full paragraphs width. What does this look like?

Reply:
really?

Reply:
![bild](https://github.com/user-attachments/assets/da46997f-2f33-444c-95d3-dfe18f18db53)
Current look for reference.